traffic_portal/app/src/common/modules/table/roleUsers/table.roleUsers.tpl.html

```diff
@@ -47,7 +47,7 @@
                 <td data-search="^{{::u.username}}$">{{::u.username}}</td>
                 <td data-search="^{{::u.email}}$">{{::u.email}}</td>
                 <td data-search="^{{::u.tenant}}$">{{::u.tenant}}</td>
-                <td data-search="^{{::u.rolename}}$">{{::u.rolename}}</td>
+                <td data-search="^{{::u.roleName}}$">{{::u.roleName}}</td>
```

Comment:
Just as a reminder, this update shouldn't be required now that the roleName -> rolename bug has been fixed, right?

Reply:
good catch. thanks.